deploy/crds/crd-clusterissuers.v1beta1.yaml

```diff
@@ -75,6 +75,9 @@ spec:
                     email:
                       description: Email is the email address to be associated with the ACME account. This field is optional, but it is strongly recommended to be set. It will be used to contact you in case of issues with your account or certificates, including expiry notification emails. This field may be updated after the account is initially registered.
                       type: string
+                    enableNotAfterDate:
```

Comment:
I am +1 to having this in general, and allowing it to be configurable in this way. That said, I'm not sure if:

1) we should have a distinct struct here containing 'certificaterequest adjustments' (or order options, whatever they are called :))
2) the naming of the field (perhaps `honourDurationField`? Though I don't like 'honour' as a word personally.. 🙄)

My reasoning for `duration` vs `notAfter` in the field name is that users of the *Certificates* API never see/configure `notAfter`.

Reply:
Agreed here! 
the struct techniclally does not touch the CR only the Order resource as the CR always has the duration whereas the order does not as it is ACME specific. 

thinking a bit.... requestDuration, sendDuration....